sdks/python/apache_beam/runners/direct/transform_evaluator.py

```diff
@@ -669,7 +669,7 @@ def start_bundle(self):
     assert len(self._outputs) == 1
     self.output_pcollection = list(self._outputs)[0]
 
-    # The output type of a GroupByKey will be KV[Any, Any] or more specific.
```

Comment:
Are you planning to convert KV to Tuple, or should we keep supporting the convenience types `KV` and `WindowedValue`?

Reply:
KV is literally just an alias for Tuple, probably not worth its weight to keep outside of the Typing module. WindowedValue should be doable as a Generic type, but it's mostly internal, so deciding on it's final state can wait as well.